README.md

```diff
@@ -1,381 +1,484 @@
-
-<p style="text-align:center;" align="center"><a href="https://meshery.io"><picture>
- <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/meshery-logo-light-text-side.svg">
- <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/meshery-logo-dark-text-side.svg">
-<img src="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/readme/meshery-logo-dark-text-side.svg"
-alt="Meshery Logo" width="70%" /></picture></a><br /><br /></p>
-<p align="center">
-<a href="https://hub.docker.com/r/meshery/meshery" alt="Docker pulls">
-  <img src="https://img.shields.io/docker/pulls/layer5/meshery.svg" /></a>
-<a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+org%3A+label%3A%22help+wanted%22+" alt="GitHub issues by-label">
```

Comment:
Thanks, @praduman8435.

I don't know that this relocation is desirable, @Yashsharma1911

Reply:
Hi! @leecalcote
Thank you for your feedback

If you think we should add or take out anything specific, just tell me. I'll check again to see if moving stuff around in the README could be better and make it more desirable. Your input is essential in making sure the README helps everyone

Reply:
@Yashsharma1911 opened the issue. I'll let him offer specifics. I believe that he already did offer much guidance in the open issue.

If this isn't an area of comfort and you need help, @praduman8435, I don't know that there's any to offer you that wouldn't make sense for another contributor to just execute themselves. In other words, of the time that I were to take explaining it, I would do better to just execute the request. 

Should we reassign the issue or can you handle it?

Reply:
Apologies if there were any misunderstanding, I'm comfortable handling this myself. Soon I will update the PR